graphql/util.go

```diff
@@ -383,3 +385,84 @@ func ConvertDBTasksToGqlTasks(tasks []task.Task, baseTaskStatuses BaseTaskStatus
 	}
 	return taskResults
 }
+
+type Modifications struct {
```

Comment:
Since this will be used by other packages, I think it needs a more specific name. Something like "VersionModifications"

Reply:
done